### PR TITLE
Writing JSON data Tutorial - C# Language Update

### PR DESCRIPTION
**Pull request by Srainyyyy**
I added c# OOP statements to the sample code and made sure that each code block was added. The code running screenshot is as follows:
Writing JSON Data to a File
0d8125c518b0e584f1b55dd47ee89d9e
Modifying Existing JSON Data
d8b0ec6f8b3dac3c1a706f4e5f7b09c5

### CODE REVIEW

**Review by kay-kaushik — COMMENTED** [review]
image

As you can see in the code snippets , the new game data argument has been written in snake case and camel case. For better readability, I would suggest following one naming convention.

File: src/content/docs/guides/JSON/2-json_writing.mdx [hunk 1]
```diff
@@ -95,6 +126,8 @@ json_set_object(new_game_data, "screenSize", screen_size_data);
 </TabItem>
 
 <TabItem label="C#">
+<Tabs syncKey="csharp-style">
+<TabItem label="Top-level Statements">
 
 ```csharp
 Json screenSizeData = CreateJson();
```
Comment (kay-kaushik) [inline src/content/docs/guides/JSON/2-json_writing.mdx:133]: I think showing the use of `using` static `SplashKitSDK.SplashKit;` in top level code could be a good idea which will make it easier to understand. Its just a suggestion.

**Comment by kay-kaushik** [comment]
Everyting else looks good.

File: src/content/docs/guides/JSON/2-json_writing.mdx (hunk 1 above)
Comment (Srainyyyy) [reply src/content/docs/guides/JSON/2-json_writing.mdx:133]: We discussed this statement earlier, and for the sake of compatibility with other tutorials in this series, we do not use static statements

**Review by Jinx-means-jinx — APPROVED** [review]
Testing the code for fill_triangle_on_bitmap
C++:
Test-Fill-trangle-on-batman
C#:
Test-Fill-trangle-on-batman-c#
Code working with no Error.
Overall Look great!

**Pull request merged**